py4DSTEM/process/calibration/braggvectors.py

```diff
@@ -100,6 +101,10 @@ def center_braggpeaks(
     Returns:
         (PointListArray): the centered Bragg peaks
     """
+
+    if isinstance(braggpeaks, BraggVectors):
+        braggpeaks = braggpeaks.vectors_uncal
+
```

Comment:
This has to go.  These functions change the underlying data in the PointListArray, and braggpeaks.vectors_uncal should *never* be modified.  It needs to be copied first, and the output passed in to this function, as when you run braggpeaks.calibrate().

Reply:
OK I think I misunderstood how this gets called, since I was adapting an old notebook. So just to be sure, this function should never actually get a `BraggVectors` object as an argument? And thus I should just drop these three lines fully?

Reply:
Yup, you got it.  Really this function should only be called internally through `BraggVectors.calibrate()`, which will handle first copying the underlying uncalibrated PLA, modifying the new, copied PLA with these functions, and then replacing the calibrated vectors attribute with this new PLA.